### PR TITLE
Make TNF image pass preflight

### PR DESCRIPTION
**Pull request by sebrandon1**
This is part 1.  I just want to get the `tag1` image building and pushing the quay then I can switch the actual tnf image over to use `tag1` or whatever we want to call it.

### CODE REVIEW

File: .github/workflows/cnf-test-partner-image.yaml [hunk 1]
```diff
@@ -56,9 +57,16 @@ jobs:
           username: ${{ secrets.QUAY_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 
-      - name: 'Build and push the new `cnf-test-partner` image'
+      - name: 'Build and push the new `cnf-test-partner` latest image'
         uses: docker/build-push-action@v3
         with:
           push: true
           file: ${{ env.IMAGE_CONTAINER_FILE_PATH }}
           tags: '${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}'
```
Comment (edcdavid) [inline .github/workflows/cnf-test-partner-image.yaml:65]: Why not just add the new tage here:
tags: '${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.IMAGE_TAG }}, ${{ env.REGISTRY }}/${{ env.IMAGE_NAME }}:${{ env.SPECIFIC_IMAGE_TAG }} '

File: .github/workflows/cnf-test-partner-image.yaml [hunk 2]
```diff
@@ -56,9 +57,16 @@ jobs:
           username: ${{ secrets.QUAY_ROBOT_USERNAME }}
           password: ${{ secrets.QUAY_ROBOT_TOKEN }}
 
-      - name: 'Build and push the new `cnf-test-partner` image'
+      - name: 'Build and push the new `cnf-test-partner` latest image'
```
Comment (edcdavid) [inline .github/workflows/cnf-test-partner-image.yaml:60]: latest image -> image

**Review by edcdavid — APPROVED** [review]

File: .github/workflows/cnf-test-partner-image.yaml (hunk 2 above)
Comment (sebrandon1) [reply .github/workflows/cnf-test-partner-image.yaml:60]: Don't think I should put latest here?

File: .github/workflows/cnf-test-partner-image.yaml (hunk 1 above)
Comment (sebrandon1) [reply .github/workflows/cnf-test-partner-image.yaml:65]: Does that work?  I'll give it a try.

File: .github/workflows/cnf-test-partner-image.yaml (hunk 2 above)
Comment (sebrandon1) [reply .github/workflows/cnf-test-partner-image.yaml]: Done.

File: .github/workflows/cnf-test-partner-image.yaml (hunk 1 above)
Comment (sebrandon1) [reply .github/workflows/cnf-test-partner-image.yaml]: Done.

**Pull request merged**